recipes/qcustomplot/all/CMakeLists.txt

```diff
@@ -32,6 +32,11 @@ option(QCUSTOMPLOT_USE_OPENGL "Use OpenGL" OFF)
 
 if(QCUSTOMPLOT_USE_OPENGL)
     target_compile_definitions(${PROJECT_NAME} PUBLIC QCUSTOMPLOT_USE_OPENGL)
+	target_link_libraries(${PROJECT_NAME} PUBLIC Qt6::OpenGL)
+	if (MSVC)
+	    target_link_libraries(${PROJECT_NAME} PRIVATE OpenGL32)
+	endif()
+	#TODO: Other platforms may need other libraries linked in
```

Comment:
Not like this, put this logic in `find_package()` blocks.

- And Qt5?
- OpenGL32 should come for free from `Qt6::OpenGL` target

Reply:
Oops... Will try to improve these....